src/h2olog/h2olog.cc

```diff
@@ -251,7 +281,19 @@ int main(int argc, char **argv)
         cflags.push_back(generate_header_filter_cflag(response_header_filters));
     }
 
+    if (event_type_filters.empty()) {
+        std::copy(available_usdt_names.cbegin(), available_usdt_names.cend(), std::back_inserter(event_type_filters));
+    }
```

Comment:
```c
if (probes.empty())
  probes = available_probes;
```

Reply:
It was intended because the two has different types (std::vector vs std::unordered_set), but eventually I've decided to make them std::set to make the code simpler: 1d531dd6398773d893a167a5b7ea4fdf811e2777

I used std::vector to keep a specific order of `event_type_filter`  in `-dd` mode, but it's enough if it's std::set instead of keeping the order of user input or  `tracer->usdt_probes()`'s. Thus, I've employed std::set instead of std::unordered_set here.

Reply:
And more: fb60dca73000ba3c67684de96493264b60bbe25d